modules/thunder_updater/src/Command/GenerateThunderUpdateCommand.php

```diff
@@ -0,0 +1,314 @@
+        $this->trans('commands.generate.thunder.update.questions.update-n'),
+        $next_update_number,
+        function ($update_number) use ($last_update_number) {
+          if (!is_numeric($update_number)) {
+            throw new \InvalidArgumentException(
+              sprintf(
+                $this->trans('commands.generate.thunder.update.messages.wrong-update-n'),
+                $update_number
+              )
+            );
+          }
+          else {
+            if ($update_number <= $last_update_number) {
+              throw new \InvalidArgumentException(
+                sprintf(
+                  $this->trans('commands.generate.thunder.update.messages.wrong-update-n'),
+                  $update_number
+                )
+              );
+            }
+            return $update_number;
+          }
+        }
+      );
+
+      $input->setOption('update-n', $update_number);
+    }
+
+    // Get description from wizard.
+    if (!$description) {
+      $description = $io->ask(
+        $this->trans('commands.generate.thunder.update.questions.description'),
+        $this->trans('commands.generate.thunder.update.defaults.description')
+      );
+      $input->setOption('description', $description);
+    }
+
+    // Get success message for checklist.
+    if (!$success_message) {
+      $success_message = $io->ask(
+        $this->trans('commands.generate.thunder.update.questions.success-message'),
+        $this->trans('commands.generate.thunder.update.defaults.success-message')
+      );
+      $input->setOption('success-message', $success_message);
+    }
+
+    // Get failure message for checklist.
+    if (!$failure_message) {
+      $failure_message = $io->ask(
+        $this->trans('commands.generate.thunder.update.questions.failure-message'),
+        $this->trans('commands.generate.thunder.update.defaults.failure-message')
+      );
+      $input->setOption('failure-message', $failure_message);
+    }
+
+    // Get list of modules that are included in update.
+    $include_modules = $input->getOption('include-modules');
+    if (!$include_modules && $use_wizard_for_optional) {
+      $include_modules = $io->ask(
+        $this->trans('commands.generate.thunder.update.questions.include-modules'),
+        ' '
+      );
+      $input->setOption('include-modules', trim($include_modules));
+    }
+
+    // Get filter regex from wizard.
+    $filter_modules = $input->getOption('filter-modules');
+    if (!$filter_modules && $use_wizard_for_optional) {
+      $filter_modules = $io->ask(
+        $this->trans('commands.generate.thunder.update.questions.filter-modules'),
+        ' '
+      );
+      $input->setOption('filter-modules', trim($filter_modules));
+    }
+  }
+
+  /**
+   * Get update gook generator.
+   *
+   * @return \Drupal\thunder_updater\Generator\ThunderUpdateGenerator
+   *   Update hook generator for Thunder update.
+   */
+  protected function createGenerator() {
```

Comment:
This is not needed. You already injected ThunderUpdateGenerator into this class

Reply:
👍